apps/archive/archive.py

```diff
@@ -443,8 +444,10 @@ def deschedule_item(self, updates, original):
         :param original: original is document.
         """
         updates[ITEM_STATE] = CONTENT_STATE.PROGRESS
-        updates[PUBLISH_SCHEDULE] = None
-        updates[SCHEDULE_SETTINGS] = {}
+        # we need to convert schedule time to UTC
+        tz = pytz.timezone(original[SCHEDULE_SETTINGS]['time_zone'] or 'UTC')
+        localized = tz.localize(original['publish_schedule'].replace(tzinfo=None))
+        updates[PUBLISH_SCHEDULE] = localized.astimezone(pytz.utc)
```

Comment:
@jerome-poisson I see your point. I think the issue might be on the client then.

Afaik, when a schedule saved by user, client sends date, time and timezone to server. Server uses PUBLISH_SCHEDULE field for the raw datetime value (as given by client) and uses SCHEDULE_SETTINGS to keep the UTC values.

You can test this by saving a story with schedule (without publishing) and if you re-open the story to see how the client sets date, time and timezone values; if the problem in the client you might see the same problem there as well.